### PR TITLE
tBTC user stats on the overview page

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #490 

This PR adds the tBTC user stats on the overview page. In this card, we display user's tBTC balance and their latest 2 bridge activities. Also users can navigate to a tBTC page from this card. Here we also update the layout of the overview page.

### Main changes:
- tBTC user stats card on the overview page- display user's tBTC balance and their latest 2 bridge activities,
- refactor `BridgeActivity` component- move the `BridgeActivity` component to shared components and add a context so we can keep the same logic(like displaying empty states) in different places of the dapp (eg. overview page and tBTC page).

<details>
  <summary>Screenshots</summary>
  
obraz

</details>

### CODE REVIEW

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left some comments to look at before the merge

File: src/pages/Overview/Network/tBTCUserStats.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        New Mint
+      </ButtonLink>
+      <Divider my="6" />
+      <BodyMd mb="6">My Activity</BodyMd>
+      <BridgeActivity
+        data={_bridgeActivity}
+        isFetching={isBridgeActivityFetching}
+      >
+        <BridgeActivityData />
+      </BridgeActivity>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/pages/Overview/Network/tBTCUserStats.tsx]: 7bc0b2f

File: src/components/tBTC/BridgeActivity.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    <></>
+  )
+}
+
+const EmptyActivity: FC = () => {
+  return (
+    <>
+      <ActivityItemWrapper>
+        <Box as="span">-.--</Box>
+        <Box as="span">-.--</Box>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/components/tBTC/BridgeActivity.tsx]: 517d81f 7dbaa9b

File: src/components/tBTC/BridgeActivity.tsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+      </ActivityItemWrapper>
+      <ActivityItemWrapper>
+        <Box as="span">-.--</Box>
+        <Box as="span">-.--</Box>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/components/tBTC/BridgeActivity.tsx]: 7dbaa9b

File: src/pages/tBTC/Bridge/BridgeActivityCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  BridgeActivityEmptyHistoryImg,
+  BridgeActivityProps,
+} from "../../../../components/tBTC"
+
+export const BridgeActivityCard: FC<
+  ComponentProps<typeof Card> & BridgeActivityProps
+> = ({ isFetching, data, children, ...props }) => {
+  return (
+    <Card {...props} minH="530px">
+      <LabelSm mb="5">my activity</LabelSm>
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/BridgeActivityCard/index.tsx:16]: I see your point and I'm good with that! 👍 Resolving this conversation

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🥍

**Pull request merged**